core/files/configure_misp.sh

```diff
@@ -97,7 +97,8 @@ set_up_oidc() {
                 \"client_secret\": \"${OIDC_CLIENT_SECRET}\",
                 \"roles_property\": \"${OIDC_ROLES_PROPERTY}\",
                 \"role_mapper\": ${OIDC_ROLES_MAPPING},
-                \"default_org\": \"${OIDC_DEFAULT_ORG}\"
+                \"default_org\": \"${OIDC_DEFAULT_ORG}\",
+                \"mixedAuth\": \"${OIDC_MIXEDAUTH}\"
```

Comment:
Should this be added only when used?

Reply:
It defaults to false (previous) behavior, so indeed could be added only when needed like the `issuer`.

Reply:
changed by 3c8a63077b44523cbbc912b6a02617719a513546. should I remove it from template.env too? if so, how do users know of it's existence?